draft-ma-opsawg-schedule-yang.md

```diff
@@ -227,89 +292,193 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
 
 ~~~~
 {
-  "period-of-time": {
-    "period-start": "2023-01-01T08:00:00Z",
-    "period-duration": "P15DT05:20:00"
-  }
+  "period-start": "2023-01-01T08:00:00Z",
+  "duration": "P15DT05:20:00"
+}
+~~~~
+
+   An example of a period that starts at 2:00 A.M. in Los Angeles on November 19,
+   2023 and lasts 20 weeks:
+
+~~~~
+{
+  "period-start": "2023-11-19T02:00:00",
+  "time-zone-identifier": "America/Los_Angeles",
+  "duration": "P20W"
+}
+~~~~
+
+## The "recurrence" Grouping
+
+   The following snippet can be used to indicate a every other day recurrence for
```

Comment:
```suggestion
   The following snippet can be used to indicate a daily recurrence for
```

Reply:
There is a difference between "every other day" and "daily", by "daily", the interval value is 1, while by "every other day", we mean “every two days”, the interval value is 2.